rainier-core/src/main/scala/com/stripe/rainier/core/SBCModel.scala

```diff
@@ -0,0 +1,48 @@
+package com.stripe.rainier.core
```

Comment:
have you tried this in the test package yet?

Reply:
I always get a `[error] java.lang.RuntimeException: No main class detected.` when I put it test....by which I mean, putting it into `rainier-tests/src/test/scala/com/stripe/rainier/core` and leaving this is `package com.stripe.rainier.core`, then trying to run it with
```
sbt:rainier-tests> run
[info] Including from cache: asm-tree-6.0.jar
[info] Including from cache: asm-6.0.jar
[info] Checking every *.class/*.jar file's SHA-1.
[info] Merging files...
[warn] Merging 'module-info.class' with strategy 'discard'
[warn] Strategy 'discard' was applied to a file
[info] Assembly up to date: /Users/mio/stripe/rainier/.asm-deps/target/asmDeps-assembly-0.1.0.jar
[info] Packaging /Users/mio/stripe/rainier/rainier-core/target/scala-2.12/rainier-core_2.12-0.1.0.jar ...
[info] Done packaging.
[error] java.lang.RuntimeException: No main class detected.
```

Reply:
@mio-stripe try putting in `rainier-tests/src/main/scala/com/stripe/rainier/core` (note the `src/main`)

Reply:
🎉 thanks! Definitely should be in `src/main` :)